logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/Conversation.kt

```diff
@@ -40,6 +41,11 @@ class MembersInfo(val self: Member, val otherMembers: List<Member>)
 
 class Member(override val id: UserId) : User()
 
+sealed class MemberDetails {
```

Comment:
At the moment, it only contains info about the user.

In the future, we can expand it with things like `isAdmin` or `conversationRole`, or other stuff like this.